source/blog/_posts/2016-07-14-open-hackathon-review.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Impressions and Results of the open hackathon
+tags:
+- hackathon
+- swagathon
+- open
+- community
+
+categories:
+- dev
+
+authors: [bc]
+indexed: true
+
+github_link: blog/_posts/2016-07-14-open-hackathon-review.md
+---
+
+<img src="/blog/img/open-hackathon-2016/cody_mate.jpg" alt="" width="140" class="is-float-left" />
+Last year in august we ran our [first Open Hackathon](https://en.shopware.com/impressive-outcome-from-the-shopware-open-hackathon/) together with in Münster. It was a great success so naturally we repeated the event this year again.
```

Comment:
In August of last year, we organized our [first Open Hackathon](https://en.shopware.com/impressive-outcome-from-the-shopware-open-hackathon/) together with external developers in Münster, Germany. Since it was such an outstanding success, it was a given we'd repeat the event this year.